### PR TITLE
Add unlit button to use Nanome 1.23.2 unlit material for mesh shape

### PR DESCRIPTION
**Pull request by nezix**

### CODE REVIEW

File: nanome_msms/MSMS.py [hunk 1]
```diff
@@ -99,17 +105,20 @@ async def structure_clicked(self, dropdown, item):
...
         opacity_slider = self.menu.root.find_node("Opacity").get_content()
         probe_slider = self.menu.root.find_node("ProbeRadius").get_content()
+        unlit_btn = self.menu.root.find_node("Unlit").get_content()
 
         eye.unusable = False
         color_btn.unusable = False
         sel_only.unusable = False
         by_chain.unusable = False
         ao_btn.unusable = False
+        unlit_btn.unusable = self._nanome_version < version.parse("0.35.5")
```
Comment (nezix) [inline nanome_msms/MSMS.py:115]: There !

**Comment by nezix** [comment]
We should sync the PR merge with Nanome-lib PR https://github.com/nanome-ai/nanome-lib/pull/283 and modify this line accordingly:
https://github.com/nanome-ai/plugin-msms/pull/4/files#r826002056

File: nanome_msms/MSMS.py [hunk 2]
```diff
@@ -24,6 +27,7 @@ def start(self):
         self._msms_instances = {}
         self._msms_tasks = {}
         self.create_menu()
+        self._nanome_version = version.parse(nanome.__version__)
```
Comment (mjrosengrant) [inline nanome_msms/MSMS.py:30]: You should pin the nanome version in requirements.txt, and remove this variable from the plugin

`nanome==0.35.8`

**Review by mjrosengrant — CHANGES_REQUESTED** [review]
You should copy this file into .github/workflows, and it provides an easy way to update nanome-lib version in the future.

https://github.com/nanome-ai/plugin-chemical-interactions/blob/master/.github/workflows/bump-nanome-lib.yml

File: nanome_msms/MSMS.py (hunk 2 above)
Comment (nezix) [reply nanome_msms/MSMS.py:30]: I wanted the plugin to still work for previous versions but removing some features

**Review by mjrosengrant — APPROVED** [review]

**Pull request merged**